lua/nvim_lsp/pyls_ms.lua

```diff
@@ -45,12 +45,12 @@ local function make_installer()
       error('Unable to identify host operating system')
     end
 
-    local url = string.format("https://pvsc.azureedge.net/python-language-server-stable/Python-Language-Server-%s-x64.0.4.114.nupkg", string.lower(system))
+    local url = string.format("https://pvsc.azureedge.net/python-language-server-stable/Python-Language-Server-%s-x64.0.5.10.nupkg", string.lower(system))
     download_cmd = string.format('curl -fLo %s --create-dirs %s', install_info.install_dir .. "/pyls.nupkg", url)
 
-    if vim.fn.has('mac') or vim.fn.has('unix') then
+    if system == 'osx' or system == 'linux' then
       install_cmd = "unzip " .. install_info.install_dir .. "/pyls.nupkg -d " .. install_info.install_dir
-    elseif vim.fn.has('win32') then
+    elseif system == 'win' then
```

Comment:
the `system` variable only exists to build the URL. Would be better to avoid proliferating use of it. has() is idiomatic.

Reply:
Updated to match #72